### PR TITLE
Fix/4526 changesiderbarexplorer

### PR DESCRIPTION
**Pull request by chrlschwb**
- #4526 
SUBSCAN
svg icon used for the link

### CODE REVIEW

**Comment by dmtrjsg** [comment]
Looking good on the preview branch 👍

File: packages/ui/src/common/components/icons/symbols/ExplorerSymbol.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+
+export function ExplorerSymbol() {
+  return (
```
Comment (thesan) [inline packages/ui/src/common/components/icons/symbols/ExplorerSymbol.tsx:4]: Looks good but please use this version (it's optimized to reduce it's size):
```jsx
<svg width="26px" height="26px" viewBox="0 0 26 26" version="1.1" xmlns="http://www.w3.org/2000/svg">
  <g fill="currentColor">
    <path d="m13.8 3.21c0.733-0.733 0.733-1.92 0-2.66-0.733-0.733-1.92-0.733-2.66 0l-10.6 10.6c-0.733 0.733-0.733 1.92 0 2.66l5.31 5.31v2.08e-4c0.733 0.733 1.92 0.733 2.66 0l9.29-9.29 2.66 2.66-9.29 9.29c-0.733 0.733-0.733 1.92 0 2.66 0.733 0.733 1.92 0.733 2.66 0l10.6-10.6c0.733-0.733 0.733-1.92 0-2.66l-5.31-5.31c-0.733-0.733-1.92-0.733-2.66 0l-9.29 9.29-2.66-2.66 9.29-9.29z" opacity=".2"/>
    <path d="m3.21 11.2c0.733 0.733 0.733 1.92 0 2.66-0.733 0.734-1.92 0.734-2.66 0-0.733-0.733-0.733-1.92 0-2.66 0.733-0.734 1.92-0.734 2.66 0"/>
    <path d="m8.52 5.86c0.733 0.734 0.733 1.92 0 2.66-0.733 0.733-1.92 0.733-2.66 0-0.733-0.733-0.733-1.92 0-2.66 0.733-0.734 1.92-0.734 2.66-2.08e-4"/>
    <path d="m13.8 0.55c0.733 0.734 0.733 1.92 0 2.66-0.733 0.733-1.92 0.733-2.66 0-0.733-0.733-0.733-1.92 0-2.66 0.733-0.733 1.92-0.733 2.66 0"/>
    <path d="m8.52 16.5c0.733 0.733 0.733 1.92 0 2.66-0.733 0.733-1.92 0.733-2.66 0-0.733-0.733-0.733-1.92 0-2.66 0.733-0.733 1.92-0.733 2.66 0"/>
    <path d="m13.8 11.2c0.733 0.733 0.733 1.92 0 2.66-0.733 0.734-1.92 0.734-2.66 0-0.733-0.733-0.733-1.92 0-2.66 0.733-0.734 1.92-0.734 2.66 0"/>
    <path d="m19.1 5.86c0.733 0.734 0.733 1.92 0 2.66-0.733 0.733-1.92 0.733-2.66 0-0.733-0.733-0.733-1.92 0-2.66 0.733-0.734 1.92-0.734 2.66-2.08e-4"/>
    <path d="m13.8 21.8c0.733 0.733 0.733 1.92 0 2.66-0.733 0.733-1.92 0.733-2.66 0-0.733-0.733-0.733-1.92 0-2.66 0.733-0.733 1.92-0.733 2.66 0"/>
    <path d="m19.1 16.5c0.733 0.733 0.733 1.92 0 2.66-0.733 0.734-1.92 0.734-2.66 2.08e-4 -0.733-0.733-0.733-1.92 0-2.66 0.733-0.733 1.92-0.733 2.66 0"/>
    <path d="m24.5 11.2c0.733 0.733 0.733 1.92 0 2.66-0.733 0.734-1.92 0.734-2.66 0-0.733-0.733-0.733-1.92 0-2.66 0.733-0.734 1.92-0.734 2.66 0"/>
  </g>
</svg>
```

**Review by thesan — CHANGES_REQUESTED** [review]

File: packages/ui/src/common/components/icons/symbols/ExplorerSymbol.tsx (hunk 1 above)
Comment (thesan) [reply packages/ui/src/common/components/icons/symbols/ExplorerSymbol.tsx:4]: Also please rename the file to `SubscanSymbol.tsx`

**Review by thesan — APPROVED** [review]
Perfect thanks!

**Pull request merged**